modules/installation-mirror-repository.adoc

```diff
@@ -57,11 +92,103 @@ $ oc adm -a ${LOCAL_SECRET_JSON} release mirror \
 ----
 +
 This command pulls the release information as a digest, and its output includes
-the `imageContentSources` data that you require when you install your cluster.
+text that resembles the following sample:
+
+. Record the
+ifdef::install[]
+`imageContentSources`
+endif::install[]
+ifdef::update[]
+`ImageContentSourcePolicy`
+endif::update[]
+section from the output of the previous
+command. This information is required
+ifdef::install[]
+during {product-title} installation.
+endif::install[]
+ifdef::update[]
+when you update your {product-title} cluster.
+endif::update[]
+
+. Build the signature for the content that you mirrored and verify that it is
+signed by an official Red Hat key:
+.. Set the digest for the version to
+ifdef::install[]
+install:
+endif::install[]
+ifdef::update[]
+update to:
+endif::update[]
++
+----
+$ DIGEST="$(oc adm release info quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} | sed -n 's/Pull From: .*@//p')" <1>
+----
+<1> For `<release_version>`, specify the version number of {product-title} to
+that you mirrored content for.
+
+.. Build a signature URI:
++
+----
+$ URI="https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/${DIGEST/:/=}/<signature_name>" <1>
+----
+<1> Specify the name of the signature to store.
```

Comment:
Do we want to give folks instructions on constructing this name?  It should be `signature-1`, falling back to `signature-2`, etc. until you either get an acceptable signature ("good sig by Red Hat's f21541eb; the release is fine") or a 404 and give up ("I guess this release isn't signed by anyone I trust").

Reply:
I think I'm missing something. Isn't this just the name of the signature that you're storing? What would you be falling back to?

Reply:
Docs for signature-lookup here and here. Yes, <signature_name> is going to be the local filename, but you can't pick any filename, it needs to follow signature-<index> to match a signature that exists on our signature mirrors.